src/account/AccountFactory.sol

```diff
@@ -14,19 +14,19 @@ contract AccountFactory is Ownable {
     UpgradeableModularAccount public immutable ACCOUNT_IMPL;
     bytes32 private immutable _PROXY_BYTECODE_HASH;
     IEntryPoint public immutable ENTRY_POINT;
-    address public immutable SINGLE_SIGNER_VALIDATION;
+    address public immutable SINGLE_SIGNER_VALIDATION_MODULE_ADDRESS;
```

Comment:
For this and the env var, I think we should be safe to omit the `_ADDRESS`. In this case it's already stated in the type, and in the env vars it's adjacent to other vars of type address. It's just getting kinda wordy

Reply:
I think it's nice to be able differentiate between instances of the contract vs. the raw address just by looking at the variable name. Agree it's wordy, but I think the clarity makes it worth it. I do agree it's less important for the env var name though.

Reply:
Fixing all instances of this.